src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,43 @@
-import React from 'react';
+import { Post } from '../../types/Posts';
+import { Comment } from '../../types/Comments';
+import { User } from '../../types/Users';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+interface Props {
+  info: Post,
+  comment: Comment[],
+  user: User,
+}
+
+export const PostInfo: React.FC<Props> = ({ info, comment, user }) => {
+  const { title, body } = info;
+
+  return (
+    <>
```

Comment:
you don`t need here Fragment

Reply:
I know, I was confused because of the previous comments